clusters/app.ci/supplemental-ci-images/html-proofer_bc.yaml

```
@@ -0,0 +1,35 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: html-proofer
+  namespace: ci
+spec:
+  tags:
+    - name: "3.19.0"
+      from:
+        kind: DockerImage
+        name: "docker.io/klakegg/html-proofer:3.19.0"
+---
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+metadata:
+  namespace: ci
+  name: html-proofer-with-bash
```

```suggestion
  name: html-proofer-with-bash-qci
```

Could we move this into the existing file `html-proofer_is.yaml`?

---

updated